<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StopwatchClass</class>
 <widget class="QWidget" name="StopwatchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stopwatch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="TimeLabel">
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="TimeTable">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>Time point</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Difference</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="StartPauseButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="TimePointButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Time point</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>StartPauseButton</sender>
   <signal>clicked()</signal>
   <receiver>StopwatchClass</receiver>
   <slot>OnStartPausePress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StopButton</sender>
   <signal>clicked()</signal>
   <receiver>StopwatchClass</receiver>
   <slot>OnStopPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TimePointButton</sender>
   <signal>clicked()</signal>
   <receiver>StopwatchClass</receiver>
   <slot>OnTimePointPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnStartPausePress()</slot>
  <slot>OnStopPress()</slot>
  <slot>OnTimePointPress()</slot>
 </slots>
</ui>
